src/doc/partials/forms.hbs

```diff
@@ -19,21 +19,64 @@
         <input type="file" id="exampleInputFile">
         <p class="help-block">Example block-level help text here.</p>
       </div>
-      <div class="checkbox">
-        <label>
-          <input type="checkbox"> Check me out
-        </label>
-      </div>
-      <div class="radio">
-        <label>
-          <input type="radio" name='radio1'> Radio me out
-        </label>
+
+      <div class="form-group">
+        <p class="form-group-label">Checkbox</p>
+        <div class="checkbox">
+          <label>
+            <input type="checkbox"></input>
+            <span>Rest</span>
+          </label>
+        </div>
+
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" checked></input>
+            <span>Checked</span>
+          </label>
+        </div>
+
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" disabled="true"></input>
+            <span>Disabled</span>
+          </label>
+        </div>
+
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" id="foo"></input>
+            <span>Indeterminate</span>
+          </label>
+          <script type="text/javascript">
+              var checkbox = document.getElementById("foo");
+              checkbox.indeterminate = true;
+            </script>
```

Comment:
The tab size is set to 2 in the Sublime project file. I think we should make this consistent with OneUI. What is the tab size there? Andy, Joe, do you have a preference?